src/main/java/FlyChat/Core/FlyChat.java

```diff
@@ -0,0 +1,99 @@
+package flychat.core;
```

Comment:
This core package is quite nice currently, but I feel that it may need refactoring in future. 
For example, if theres alot more storage related files, likely need to wrap them into a separate Storage package instead!